woocommerce/woocommerce-android
# AI Product description: Add API calls

Fixes #9334, a subtask of #9214.

[device-2023-06-30-193939.webm](https://github.com/woocommerce/woocommerce-android/assets/1522856/3e1a9cca-a538-4137-95f9-cd21809def05)

This PR adds the API calls to generate product description. It seems like the chosen language is always different based on the product name and features. This needs to be fixed in FluxC, so we can go ahead with this for now.

**To test:**
1. Go to Products -> select a product -> Tap on `Write with AI` button
2. Add some features to the text box
3. Tap on `Write with AI`
4. After a few moments, notice a generated description is shown
5. Tap on the `Regenerate` button
6. After a few moments, notice a new description is shown
7. Tap on the `Copy` button
8. Notice the description is copied to the clipboard
9. Tap on the `Apply` button
10. Notice the bottom sheet is closed and the description is now applied to the product

**Note:** Please merge #9333 first.
